### PR TITLE
Fix Image Meme Generator plugin user agent

### PR DESCRIPTION
**Pull request by renshuki**
Hi,

As discussed in #905, we have the following error:
`urllib.error.HTTPError: HTTP Error 403: Forbidden`

The error is related to the user agent that is not set. I added a valid one in this PR to fix the error.

Thanks 😄

### CODE REVIEW

File: hangupsbot/plugins/image/image_memegenerator.py [hunk 1]
```diff
@@ -30,7 +30,13 @@ def meme(bot, event, *args):
             parameters.append("robot")
 
         """public api: http://version1.api.memegenerator.net"""
-        url_api = 'http://version1.api.memegenerator.net/Instances_Search?q=' + "+".join(parameters) + '&pageIndex=0&pageSize=25'
+        url_api = 'http://version1.api.memegenerator.net//Instances_Search?q=' + "+".join(parameters) + '&pageIndex=0&pageSize=25&apiKey=e12aa6f1-ed4a-4362-aeaf-1d3281e6fa29'
```
Comment (das7pad) [inline hangupsbot/plugins/image/image_memegenerator.py]: Whoops, you just published your private api-key.

File: hangupsbot/plugins/image/image_memegenerator.py [hunk 2]
```diff
@@ -39,8 +45,11 @@ def meme(bot, event, *args):
         if len(results['result']) > 0:
             instanceImageUrl = random.choice(results['result'])['instanceImageUrl']
 
-            image_data = urllib.request.urlopen(instanceImageUrl)
+            request = urllib.request.Request(instanceImageUrl,None,headers) 
+            image_data = urllib.request.urlopen(request)
```
Comment (das7pad) [inline hangupsbot/plugins/image/image_memegenerator.py]: What about using `aiohttp` to fetch the image data?

File: hangupsbot/plugins/image/image_memegenerator.py (hunk 1 above)
Comment (renshuki) [reply hangupsbot/plugins/image/image_memegenerator.py]: Wow, I definitely need some sleep... Thank you for the catch.
Any way to delete the PR?

File: hangupsbot/plugins/image/image_memegenerator.py (hunk 2 above)
Comment (renshuki) [reply hangupsbot/plugins/image/image_memegenerator.py]: Yeah, I think it's possible to do this with `aiohttp` too.

File: hangupsbot/plugins/image/image_memegenerator.py (hunk 1 above)
Comment (das7pad) [reply hangupsbot/plugins/image/image_memegenerator.py]: @renshuki The commit remains public. I suggest revoking the api-key.

**Pull request closed without merging**